lib/filterx/filterx-grammar.ym

```diff
@@ -143,6 +147,7 @@ expr
         | expr KW_TAV_EQ expr			{ $$ = filterx_comparison_new($1, $3, FCMPX_TYPE_AND_VALUE_BASED | FCMPX_EQ); }
         | expr KW_TAV_NE expr			{ $$ = filterx_comparison_new($1, $3, FCMPX_TYPE_AND_VALUE_BASED | FCMPX_NE ); }
 	| '(' expr ')'				{ $$ = $2; }
+	| fxcondition				{ $$ = $1; }
```

Comment:
If I understand correctly we need a semicolon `;` after the conditional block. Can't we just move `fxcondition` to `stmt`? I think doing so would make it possible to use them without `;`.